tests/utils_tests/test_functional.py

```diff
@@ -130,3 +131,11 @@ def test_lazy_equality(self):
 
         self.assertEqual(lazy_a(), lazy_b())
         self.assertNotEqual(lazy_b(), lazy_c())
+
+    def test_lazy_repr(self):
+        """
+        Tests that a proxy of lazy object's repr equals to the object's value
```

Comment:
docstring isn't really needed since the test is simple
do you need to use `ugettext_lazy` even though you've only modified `lazy()`?